spec/controllers/facility_reservations_controller_spec.rb

```diff
@@ -188,7 +188,7 @@
 
     context "redirect on no edit" do
       before :each do
-        @reservation.update_attribute(:canceled_at, Time.zone.now)
+        @reservation.order_detail.update_attributes(canceled_at: Time.zone.now)
```

Comment:
Rubocop flags on both `update_attribute` and `update_column`. I hesitate to change `Time.zone.now`, mostly because it's used multiple times throughout the file and I don't want to be inconsistent, but I also don't want to go on a huge style tangent in this PR.